installer/DetailsHandler.php

```diff
@@ -126,28 +126,34 @@ public function getAdminEmployeeLastName()
      * Get the organization email address
      * @return mixed
      */
-    public function getOrganizationEmailAddress()
+    public function getOrganizationEmailAddress($invalid = false)
```

Comment:
Don't pass the parameter to the function and this is a anti-pattern.
collect all the errors to a private variable in a class

Reply:
updated